app/controllers/spree/admin/base_controller.rb

```diff
@@ -110,6 +111,24 @@ def serializer(ams_prefix)
         name = controller_name.classify
         "::Api::Admin::#{prefix}#{name}Serializer".constantize
       end
+
+      def check_updated_tos_accepted
+        @terms_of_service_banner = false
+
+        return unless spree_user_signed_in?
+
+        return if accepted_tos?
+
+        @terms_of_service_banner = true
+      end
+
+      def accepted_tos?
+        file_uploaded_at = TermsOfServiceFile.updated_at
+
+        current_spree_user.terms_of_service_accepted_at.present? &&
+          current_spree_user.terms_of_service_accepted_at > file_uploaded_at &&
+          current_spree_user.terms_of_service_accepted_at < DateTime.now
```

Comment:
I'm wondering what this condition is for?

Reply:
It could only be manually updated by a developer in the console, right? Quite unlikely indeed...